app/src/main/java/com/example/tasky/agenda/presentation/agenda_detail/AgendaDetailViewModel.kt

```diff
@@ -94,6 +102,25 @@ class AgendaDetailViewModel(
             .launchIn(viewModelScope)
     }
 
+    private fun getTask(taskId: String) {
+        viewModelScope.launch {
+            taskLocalDataSource.getTask(id = taskId).firstOrNull()?.let { task ->
+                val duration = task.time - task.remindAt
```

Comment:
Should we be relying on the cached reminder value here instead? 🤔 Why is the reason we are doing this calculation by ourself ?